projects/js-packages/components/components/popover/style.scss

```diff
@@ -0,0 +1,21 @@
+.jp-popover {
+	width: 600px;
```

Comment:
This might be problematic for smaller screens?

Reply:
There is a max-width below it which would contain it for mobile devices. However, I have improved the styles after that.

Reply:
👍